test_util/helpers.py

```diff
@@ -170,6 +170,27 @@ def options(self, *args, **kwargs):
         return self.api_request('OPTIONS', *args, **kwargs)
 
 
+def is_retryable_exception(exception):
+    for ex in [requests.exceptions.ConnectionError, requests.exceptions.Timeout]:
+        if isinstance(exception, ex):
+            return True
+    return False
+
+
+class RetryCommonHttpErrorsMixin:
+    """ Mixin for ApiClientSession so that random disconnects from
+    network instability do not derail entire scripts
+    """
+    def api_request(self, *args, retry_timeout=60, **kwargs):
```

Comment:
@colin-msphere  - the readability of this code comes from  the context. The default unit of the `@retrying` library works with is `milliseconds`. So, we multiply by `1000` for the seconds unit.

That said, I am in favor of more explicit units too. My preference will be in comments rather than in variable in names.Or using some sort of Units Library, [like this](https://github.com/twitter/commons/blob/master/src/python/twitter/common/quantity/__init__.py#L22)

Here, if we rename the variable to having `_UNIT`, it might stand out as an odd duck.